client/verta/tests/unit_tests/registry/test_registered_model_version.py

```diff
@@ -89,8 +91,7 @@ def model_ver_proto(
     return msg
 
 
-@patch.object(RegisteredModelVersion, "_refresh_cache", return_value=None)
-@hypothesis.given(
+@given(
```

Comment:
Apparently `@patch.object` interferes with `@hypothesis.given`, so I moved them into the test bodies as `with` contexts.